test/handlebars/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const describeFolder = require('@quarterto/mocha-describe-folder');
+
+describeFolder('handlebars tests', function () {
+	this.timeout(5000);
```

Comment:
What happens without this line (i.e. can we do without it)?

Reply:
yes, removed it :)